apps/web/components/shared/kyc-4-upload.tsx

```
@@ -0,0 +1,531 @@
+  useEffect(() => {
+    if (previewUrl) {
+      return () => URL.revokeObjectURL(previewUrl);
+    }
+  }, [previewUrl]);
+
+  const isValidFileType = useCallback((file: File): boolean => {
+    const validTypes = ["image/png", "image/jpeg", "image/jpg"];
+    return validTypes.includes(file.type);
+  }, []);
+
+  const handleDrop = useCallback(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: toast

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)